test/unit/gif.js

```diff
@@ -98,4 +98,28 @@ describe('GIF input', () => {
       sharp().gif({ delay: [65536] });
     });
   });
+
+  it('should work with streams when only animated is set', function (done) {
```

Comment:
Not all test environments will have *magick available - please can you wrap these two GIF output tests with:
```js
if (sharp.format.magick.output.buffer) { ... }
```